packages/block-library/src/gallery/block.json

```diff
@@ -41,7 +41,10 @@
 			"default": [ ]
 		},
 		"columns": {
-			"type": "number"
+			"type": "number",
+			"default": 3,
+			"min": 1,
```

Comment:
JSON schema uses "minimum" and "maximum" https://json-schema.org/understanding-json-schema/reference/numeric.html#integer, I guess should use the same properties.

Reply:
Min and max are used elsewhere in the code base. 
https://github.com/WordPress/gutenberg/blob/5494bc57fc77de6cada4295823a3fb047e168f89/packages/block-library/src/column/block.json#L10-L11

Reply:
The rest of code uses minimum and maximum 
https://github.com/WordPress/WordPress/blob/master/wp-includes/rest-api.php#L1227-L1267

Reply:
Huh, this hadn't occurred to me. I don't know that our JavaScript implementation enforces this validation. Generally, if the idea is to follow JSON Schema, we should probably conform to the proper names. I suppose we'd need to alias existing values to avoid breaking changes.

It seems like something we might want to consider addressing separate from this pull request, however.

Reply:
@aduth does it mean that in this PR we should use minimum and maximum, and as a follow up we should change the column block (and have a proper alias for back-compatibility)?
cc: @spacedmonkey

Reply:
> @aduth does it mean that in this PR we should use minimum and maximum, and as a follow up we should change the column block (and have a proper alias for back-compatibility)?

Yes, let's conform to the intended names here (`"minimum"`, `"maximum"`), regardless of consistency or validation support elsewhere.

Reply:
Follow-up pull request including the typo correction and numeric ranges validation: #19433